torchmetrics/text/wer.py

```diff
@@ -0,0 +1,78 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import List
+
+from torchmetrics.metric import Metric
+from torchmetrics.utilities.imports import _module_available
+
+_JIWER_AVAILABLE: bool = _module_available("jiwer")
+
+try:
+    from jiwer import compute_measures
+except ImportError:
+    compute_measures = 0
+    Exception("Jiwer not found")
```

Comment:
Should we change this to 


```suggestion
if _JIWER_AVAILABLE:
    from jiwer import compute_measures
```